### PR TITLE
DOCS-7600: CSM left nav updates

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

DOCS-7600

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/misconfigurations/custom_rules.md [hunk 1]
```diff
@@ -34,17 +34,17 @@ To view the list of the available resource types for your custom rules, see [Clo
 
 To clone a rule:
 
-1. Find the rule you want to copy one of the following ways:
-   - Navigate to [**Security** > **Configuration** > **Cloud Security Management** > **Compliance Rules**][1]. Select a rule you want to copy to open its details page.
-   - Navigate to [**Security** > **Cloud Security Management** > **Explorer** > **Misconfigurations**][2]. Select a misconfiguration to open its details, and select **Edit Rule** from the **Rule** menu.
+1. Find the rule you want to copy by doing one of the following:
+   - Navigate to the [**Misconfigurations Rules**][1] page. Select a rule you want to copy to open its details page.
+   - Navigate to the [**Misconfigurations Explorer**][2]. Select a misconfiguration to open its details, then select **Edit Rule** from the **Rule** menu.
```
Comment (estherk15) [inline content/en/security/cloud_security_management/misconfigurations/custom_rules.md:39]: Ignore if incorrect. I see the **Edit Rule** as a standalone button
```suggestion
   - Navigate to the [**Misconfigurations Explorer**][2]. Select a misconfiguration to open its details, then select **Edit Rule**.
```

File: content/en/security/detection_rules/_index.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ To [create custom rules](#create-detection-rules), you can clone the default rul
 
 ## Search and filter detection rules
 
-To view out-of-the-box and custom detection rules in Datadog, navigate to the [**Security** > **Configuration**][15] page. Rules are listed on separate pages for each product (Application Security, Cloud Security Management, and Cloud SIEM).
+To view out-of-the-box and custom detection rules in Datadog, navigate to the [**Security Settings**][15] page. Rules are listed on separate pages for each product (Application Security, Cloud Security Management, and Cloud SIEM).
```
Comment (estherk15) [inline content/en/security/detection_rules/_index.md:51]: This link sends me to https://app.datadoghq.com/security/configuration/notification-rules
which doesn't match the Cloud SIEM image, should it link to https://app.datadoghq.com/security/configuration/siem/rules?query=type%3A%28log_detection%20OR%20signal_correlation%29&deprecated=hide&groupBy=tactic&sort=rule_name ?

**Review by estherk15 — APPROVED** [review]
One question on an app link, looks good!

File: content/en/security/cloud_security_management/misconfigurations/custom_rules.md (hunk 1 above)
Comment (drichards-87) [reply content/en/security/cloud_security_management/misconfigurations/custom_rules.md:39]: Good catch!

**Pull request merged**